api/ls_ganap/main_events/models.py

```diff
@@ -102,14 +102,11 @@ class Event(SoftDeletionModel):
 	created_at = models.DateTimeField(auto_now_add=True)
 	updated_at = models.DateTimeField(auto_now=True)
 	is_accepted = models.BooleanField(default=False)
-	poster_url = models.URLField()
+	poster_url = CloudinaryField('image')
```

Comment:
![screen shot 2018-08-10 at 10 37 15 pm 2](https://user-images.githubusercontent.com/20733806/43964198-d76b4f2a-9cee-11e8-8750-ddd300f5b2b0.png)

should come out as poster_url